scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -311,6 +311,10 @@ if [[ -n "${_AIRFLOW_WWW_USER_CREATE=}" ]] ; then
     create_www_user
 fi
 
+if [[ -n "${_PIP_ADDITIONAL_REQUIREMENTS=}" ]] ; then
+    pip install --no-cache-dir --user "${_PIP_ADDITIONAL_REQUIREMENTS=}"
+fi
+
```

Comment:
What is the status of the warning? Should we add it or not?

Reply:
Ah. I forgot about it.. Sorry. Adding it now.

Reply:
Added! Thanks for reminding me @mik-laj !

Reply:
I think it was lost during rebase :(